.github/workflows/ruby.yml

```diff
@@ -18,27 +18,21 @@ env:
   JRUBY_OPTS: "--dev"
 
 jobs:
-  test-ubuntu:
-    runs-on: ubuntu-latest
+  test-windows:
+    runs-on: windows-latest
```

Comment:
https://github.com/ruby/setup-ruby-pkgs

This link mentioned windows-2022 which can be tried, I guess?

Reply:
I'll see if I can get childprocess running CI on Actions.